modules/frontend/config.go

```diff
@@ -28,19 +27,31 @@ type Config struct {
 
 type SearchConfig struct {
 	Sharder SearchSharderConfig `yaml:",inline"`
+	SLO     SLOConfig           `yaml:",inline"`
 }
 
 type TraceByIDConfig struct {
 	QueryShards int           `yaml:"query_shards,omitempty"`
 	Hedging     HedgingConfig `yaml:",inline"`
+	SLO         SLOConfig     `yaml:",inline"`
```

Comment:
It's possible to configure `throughput_slo` for traces by ID, but it's unused.  Should we make the slo logic consistent between endpoints by moving this block to a shared function?  Also thinking to not require both settings, allow either or both to be set.
```go
// only record metric when it's enabled and within slo
 	if s.sloCfg.DurationSLO != 0 && s.sloCfg.ThroughputSLO != 0 {
 		if reqTime < s.sloCfg.DurationSLO || throughput > s.sloCfg.ThroughputSLO {
 			// query is within SLO if query returned 200 within DurationSLO seconds OR
 			// processed ThroughputSLO bytes/s data
 			sloSearchCounter.WithLabelValues(tenantID).Inc()
 		}
 	}
```